css/homepage-styles.css

```diff
@@ -0,0 +1,76 @@
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.header-container a {
+  margin-right: 16px;
+  margin-top: 19px;
+}
+
+.headline {
+  margin: 30px 16px 17px 16px;
+  display: flex;
+  flex-direction: column;
+}
+
+.info-container > h1 {
```

Comment:
- [ ] I highlighted this piece of code in order to give an inline comment, not that this is where the problem is coming from. I noticed there is a little difference between your design and the Figma template design you are working on and I would advise you to fix it. Your design is currently missing the background image at the bottom of the `start collaboration` button, kindly add it.

Your Design 👇 

![bamfo](https://user-images.githubusercontent.com/78634841/219876151-fb358447-8195-48bd-aad7-3f165491aa43.png)

Figma Design 👇 

![temp5-mob](https://user-images.githubusercontent.com/78634841/219876176-999ac7b6-a012-43d2-83fd-e45c1021000f.png)